hyperspy/learn/rpca.py

```diff
@@ -238,10 +239,11 @@ def orpca(X, rank, fast=False,
     lambda2 : None | float
         Sparse error regularization parameter.
         If None, set to 1 / sqrt(nsamples)
-    method : None | 'CF' | 'BCD' | 'SGD'
+    method : None | 'CF' | 'BCD' | 'SGD' | 'MomentumSGD'
```

Comment:
We follow the [numpy docstring style](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) so `{None, 'CF',  'BCD',  'SGD'} instead of `None | 'CF' | 'BCD' | 'SGD'`